src/React.Tests/Core/ReactComponentTest.cs

```diff
@@ -64,6 +64,25 @@ public void RenderHtmlShouldWrapComponentInDiv()
 		}
 
 		[Test]
+		public void RenderHtmlShouldNotRenderComponentHTML()
+		{
+			var environment = new Mock<IReactEnvironment>();
+			environment.Setup(x => x.Execute<bool>("typeof Foo !== 'undefined'")).Returns(true);
+			environment.Setup(x => x.Execute<string>(@"React.renderToString(React.createElement(Foo, {""hello"":""World""}))"))
+				.Returns("[HTML]");
+			var config = new Mock<IReactSiteConfiguration>();
+
+			var component = new ReactComponent(environment.Object, config.Object, "Foo", "container")
+			{
+				Props = new { hello = "World" }
+			};
+			var result = component.RenderHtml(true); //renderContainerOnly = true
```

Comment:
Use named arguments for boolean parameters. `component.renderHtml(renderContainerOnly: true)`